src/TLuaInterpreter.cpp

```diff
@@ -5110,7 +5101,8 @@ int TLuaInterpreter::playSoundFile( lua_State * L )
     {
         sound.replace('\\', "/");
     }
-    mudlet::self()->playSound( sound, soundVolume );
```

Comment:
Yep.  In most cases the user/scripter won't pay any attention, but if you are using the command from the "command line" it is nice (IMHO) to get confirmation that the command went through Okay.  It also work better (also IMHO) if you ARE checking for errors by checking for NOT a `nil + error message` because the absence of any wanted return value from a Llua command __is a nil__ or at least that is what the variable waiting for a value gets.  e.g. : 

    function someRandomFunction()
    -- stuff
    local x = 1
    local y = "two"
    -- stuff that doesn't change x or y
    return x,y
    end

then if we use this:

    local x,y,z = someRandomFunction()

then x will be 1, y will be "two" and z will be _nil_ as I understand Lua {which is not as good as some others :wink:}.

As for a failure to find/play the sound file given; that might be a suitable case for a new "sysSoundEvent" TEvent type which might include other "callback" type features such as:
* notification of reaching the end (or some %age point in the sound playback) of a media file
* a time-out related change in the number of `QMediaPlayers` if we decide to cull them {as this series of commits only ever increases them with no ability to get rid of them if not needed}
* notifications of QMediaPlayer errors that arise after the request was made to play a file

These are just outline thoughts and not something for immediate action, more likely the "let's think about this at some idle moment" pile...